DigitalLearningSolutions.Web.Tests/ViewComponents/CurrentFiltersViewComponentTests.cs

```diff
@@ -68,6 +68,9 @@ public void CurrentFiltersViewComponent_selects_expected_filters_to_display()
             var expectedFilterViewModel = new CurrentFiltersViewModel(
                 expectedAppliedFilters,
                 searchString,
+                null,
+                null,
+                10,

```

Comment:
Minor, but do we want to change these, and their corresponding inputs, to something non-null just to make sure the model properties are populated?